mxd/backend-service/src/main/java/org/eclipse/mxd/service/Impl/ContentServiceImpl.java

```diff
@@ -0,0 +1,165 @@
+
+/*******************************************************************************
+ * Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *    Contributors:Ravinder Kumar
+ *    Backend-API and implementation
+ *
+ ******************************************************************************/
+
+package org.eclipse.mxd.service.Impl;
+
+import jakarta.ws.rs.core.MediaType;
+import org.eclipse.mxd.model.*;
+import org.eclipse.mxd.repository.ContentsRepository;
+import org.eclipse.mxd.repository.Impl.ContentsRepositoryImpl;
+import org.eclipse.mxd.service.ContentService;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.ws.rs.core.Response;
+import jakarta.ws.rs.core.UriInfo;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+
+
+@ApplicationScoped
+public class ContentServiceImpl implements ContentService {
+
+    private static final Logger logger = Logger.getLogger(ContentServiceImpl.class.getName());
+
+    private final ContentsRepository contentServiceRepository = new ContentsRepositoryImpl();
+
+    @Override
+    public Response getAll(UriInfo uriInfo) {
+        try {
+            List<ContentsRequest> contentResponse = this.contentServiceRepository.getAllAssets();
+            if (contentResponse != null && !contentResponse.isEmpty()) {
+                List<ContentModelResponse> contentsModels = new ArrayList<ContentModelResponse>();
+                for (ContentsRequest contentsRequest : contentResponse) {
+                    JsonNode jsonNode = new ObjectMapper().readTree(contentsRequest.getAsset());
```

Comment:
Create `ObjectMapper` once and resuse it. Or create a Singleton class to return a same instance everywhere.

Reply:
used object mapper